components/dashboard/cards/testimonial.tsx

```diff
@@ -0,0 +1,67 @@
+import { useDispatch } from 'react-redux'
+import { AppDispatch } from '@/lib/redux/store'
+import { setTestimonials, setDisplay } from '@/lib/redux/slices/testimonials'
+import { StyledTestimonialCard, StyledTestimonialInput } from './StyledCards'
+import { Close } from '@mui/icons-material'
+import { Button } from '@mui/material'
+import { useState } from 'react'
+import { createTestimonials } from '@/lib/db/cms/testimonials'
+
+const TestimonialCard = ({ field }: any) => {
+    const dispatch = useDispatch<AppDispatch>()
+    const [isFlipped, setIsFlipped] = useState(false)
+    const { display, email, id, name, rating, review } = field
+
+    const handleDisplayChange = (event: {
+        target: { id: string; checked: boolean }
+    }) => {
+        const { id, checked } = event.target
+        dispatch(setDisplay({ id, checked }))
+    }
+
+    const toggleDeletionView = event => {
```

Comment:
`event` is never used. no need to pass it in if it's never used.

Also, one liner this.
`const toggleDeletionView = () => setIsFlipped(!isFlipped)`

Also, since it's a function we want to name them with `handle` or `on`. `onToggleDeleteView` or `handleToggleDeleteView` to help denote it is a function triggering an action